src/main/java/uk/gov/ons/ctp/response/action/client/CaseSvcClientService.java

```diff
@@ -44,7 +45,9 @@
   @Qualifier("caseSvcClient")
   private RestUtility restUtility;
 
-  @Autowired private ObjectMapper objectMapper;
+  @Qualifier("customObjectMapper")
```

Comment:
why did this change? Seems unrelated to the PR

Reply:
@NickGrantONS Intellij was complaining about not being able to autowire the object mapper.

Reply:
I'd prefer it removed - we should let spring use autowiring wherever possible